### ISSUE TITLE
Duplicates in Z0 decays

### ISSUE THREAD

**Issue by ntiltmann** [QuarkNet-HEP/opal-masterclass#16]
I am currently also working with the OPAL dataset, especially the Z0 events. Actually, there are less than 1000 events in total, due to some duplicates. These are two events numbers on the page leading to the same .gif file.

In detail, these are:
Events 117, 366 -> z5305_9885.gif
Events 155, 990 -> z5305_24287.gif
Events 200, 370 -> z5293_19282.gif
Events 622, 987 -> z5293_8046.gif
Events 951, 953 -> z5305_10152.gif
Events 952, 954 -> z5305_10307.gif

So in total, there are 997 unique events available. However, I am currently still unsure, if the duplicate events should still be counted twice in order to get the correct branching ratios in the end. I hope this helps...

**Comment by tpmccauley** [comment]
Well spotted. This is also the case in the original masterclass. I imagine they were just counted twice in the original masterclass. 

I should point out that for three event numbers there were "extra" events. See https://github.com/QuarkNet-HEP/opal-masterclass/issues/10

Perhaps use these 3 "extra" events to replace 3 of the duplicates specified in this issue? Probably one would have to have a look and replace like-for-like.

One option, the one chosen for https://github.com/QuarkNet-HEP/opal-masterclass/issues/10 is to leave as-is and see how the results shake out after people start using it.

**Comment by tpmccauley** [comment]
I went through all Z events and found as well that 655 and 656 are the same

**Comment by tpmccauley** [comment]
I had the same answers for each of the duplicates. All expect one pair seem to be qq events. One pair looks to be tautau (622:987)

**Comment by ntiltmann** [comment]
I would agree with all events events being qq aside from the 622:987 you mentioned. 

The doubled events 655:656 should not make that much of a difference since they are to separate .gif files. I mean, they are undoubtedly the same events, but this does not change the overall number of image files. Or am I missing something here?
